Makefile

```diff
@@ -201,6 +206,10 @@ container-target-allocator-push:
 container-target-allocator:
 	docker buildx build  --load --platform linux/${ARCH} -t ${TARGETALLOCATOR_IMG} cmd/otel-allocator
 
+.PHONY: container-remote-configuration
+container-remote-configuration:
```

Comment:
Missing `container-remote-configuration-push`

Reply:
I'm leaving all CI/CD around pushing this container to a separate PR to make it easier to review that component. This goes with an E2E test as well as we don't have a CRD for this yet.